tests/test_agents.py

```
@@ -447,36 +437,31 @@ def test_set(self, fix_AgentsDF: AgentsDF):
+            original_order_0 = agents._agentsets[0].df["unique_id"].to_list()
+            original_order_1 = agents._agentsets[1].df["unique_id"].to_list()
             agents.shuffle(inplace=True)
             if (
-                original_order_0 != agents._agentsets[0].agents["unique_id"].to_list()
-                and original_order_1
-                != agents._agentsets[1].agents["unique_id"].to_list()
+                original_order_0 != agents._agentsets[0].df["unique_id"].to_list()
+                and original_order_1 != agents._agentsets[1].df["unique_id"].to_list()
             ):
                 return
         assert False
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)